e2e/scripts/actions/persistent_overlay.test

```diff
@@ -0,0 +1,49 @@
+#!/singularity/test
+# persistent overlay tests
+
+source common.func
+
+# check overlay is supported, or skip those tests
+check-overlayfs
+
+# check command presence
```

Comment:
Another great check for tests. Verify commands which we use are actually available instead of assuming they are! Great!